packages/realm-web/src/App.ts

```diff
@@ -202,6 +202,18 @@ export class App<
     credentials: Credentials,
     fetchProfile = true,
   ): Promise<User<FunctionsFactoryType, CustomDataType>> {
+    if (credentials.reuse) {
+      // TODO: Consider exposing providerName on "User" and match against that instead?
+      const existingUser = this.users.find((user) => user.providerType === credentials.providerType);
```

Comment:
If there's an existing email/password user, and a *new* email/password user attempts a login, won't this also deem the new one as "existing"?

Reply:
Correct, and currently anonymous is the only credential supporting the `reuse` property.

Reply:
Ah okay!